mne/preprocessing/xdawn.py

```diff
@@ -295,31 +285,31 @@ def fit(self, epochs, y=None):
             self.patterns_[eid] = linalg.inv(evecs.T)
 
         # store some values
-        self.ch_names = epochs.ch_names
+        self.ch_names = info['ch_names']
```

Comment:
It does. See [info](https://github.com/mne-tools/mne-python/blob/master/mne/io/meas_info.py#L66)